.golangci.yaml

```diff
@@ -9,3 +9,5 @@ linters:
 issues:
   max-issues-per-linter: 0
   max-same-issues: 0
+  exclude:
+  - SA1019 # temporarily ignore deprecation warnings
```

Comment:
Temporarily silence the linter for deprecations, because some k8s workqueue interfaces are deprecated.

Will re-enabled once we switched to the new interfaces, but this needs to be a separate change.